### DISCUSSION TITLE
Amethyst 0.19.0 Beta

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1452] [Announcements]
# Notable Changes
* Inclusion of exceptions in custom layout logging. Thanks, @DavidDTA!
* Addition of optional commands for spaces 11 and 12. Thanks, @timvisher!
* Fix an issue causing windows to "stick" when trying to throw between screens.
* Fix an issue causing throwing to spaces left and right to throw to the wrong space.
* Fix an issue that could fail to track applications if they did not launch quickly enough.
* Fix an issue preventing management of Parallels Coherence Mode windows (along with a variety of similar ones).
* A variety of stability improvements to prevent a common crash.

## What's Changed
* Add missing .yml extension in XDG config path by @jabkoo in https://github.com/ianyh/Amethyst/pull/1404
* Start a Troubleshooting doc by @ianyh in https://github.com/ianyh/Amethyst/pull/1406
* Only trigger reflow after a screen move has happened by @ianyh in https://github.com/ianyh/Amethyst/pull/1407
* Use global space indices for left and right by @ianyh in https://github.com/ianyh/Amethyst/pull/1408
* Treat yaml Null as nil by @ianyh in https://github.com/ianyh/Amethyst/pull/1412
* Update Black Girls CODE donation link by @glibsm in https://github.com/ianyh/Amethyst/pull/1414
* Bundle update by @ianyh in https://github.com/ianyh/Amethyst/pull/1423
* Log JavaScript exceptions in custom layouts by @DavidDTA in https://github.com/ianyh/Amethyst/pull/1428
* Add support for throwing to spaces 11 and 12 by @timvisher in https://github.com/ianyh/Amethyst/pull/1433
* Update to swift 5 by @ianyh in https://github.com/ianyh/Amethyst/pull/1437
* Improvements to manageable applications determination by @ianyh in https://github.com/ianyh/Amethyst/pull/1439
* Do not trigger changes when trying to toggle float on an unmanageable window by @ianyh in https://github.com/ianyh/Amethyst/pull/1440
* Fix a bunch of warnings by @ianyh in https://github.com/ianyh/Amethyst/pull/1443
* Update Silica for raise change by @ianyh in https://github.com/ianyh/Amethyst/pull/1445
* Monitor application launch completion by @ianyh in https://github.com/ianyh/Amethyst/pull/1446
* Just turn off file and type length lint rules by @ianyh in https://github.com/ianyh/Amethyst/pull/1448
* Dependency updates by @ianyh in https://github.com/ianyh/Amethyst/pull/1449
* Update Silica for run loop source change by @ianyh in https://github.com/ianyh/Amethyst/pull/1450

## New Contributors
* @jabkoo made their first contribution in https://github.com/ianyh/Amethyst/pull/1404
* @DavidDTA made their first contribution in https://github.com/ianyh/Amethyst/pull/1428
* @timvisher made their first contribution in https://github.com/ianyh/Amethyst/pull/1433

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.18.0...v0.19.0b

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.19.0b'>Amethyst 0.19.0 Beta</a>.</em>

**Comment by timvisher** [comment]
Very excited to see this release! :)

I got the beta installed and while I'm able to bind the throw commands to spaces 11 and 12 they don't seem to do anything. I tried to use [Shortcut Detective](https://www.irradiatedsoftware.com/labs/) to see whether something else was capturing the keybinding but it showed nothing using it while showing Amethyst properly for the other bindings. It also doesn't seem to matter what I try for the bindings (although for those following along at home I tried '⌘⌥^F11', '⌘⌥⇧F11', and '⌘⌥F11'. My usual bindings are '⌘⌥^F11'.

**Reply by timvisher** [reply]
I also tried binding the throw right and left commands to see if it was something about Amethyst throwing to those spaces at all and moving the window to space 10 and throwing to 11 and 12 that way worked fine.

**Reply by timvisher** [reply]
I also get a system beep when pressing either binding.

**Reply by ianyh** [reply]
Dumb oversight on my part. We added the handling, but the binding still only loops to the first 10. I can fix it tomorrow, but it's in HotKeyMananger.swift if you want to take a look at it.

**Reply by timvisher** [reply]
Maybe maybe https://github.com/ianyh/Amethyst/pull/1453? :)

🙏 glob for betas, amiright? ;P

**Reply by timvisher** [reply]
Don't see a discussion for Beta 2 but I wanted to say that i got it installed and we're golden from my end. ٩( ᐛ )و
